packages/manager/modules/pci/src/components/project/users/translations/Messages_de_DE.json

```diff
@@ -3,6 +3,7 @@
   "pci_projects_project_users_username_label": "Benutzername",
   "pci_projects_project_users_description_label": "Beschreibung",
   "pci_projects_project_users_role_label": "Rollen",
+  "pci_projects_project_users_createdAt_label": "Date de création",
```

Comment:
For translations, we only add/edit text for FR (French). The translations for the rest of the languages are to be generated via a translation request